app/retail/templatetags/to_date.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+
+import dateutil
+
+register = template.Library()
+
+@register.filter(name='to_date')
```

Comment:
would be able to reuse 
date_fromisoformat / tweak it as opposed to creating a new filter ?